fuse_constraints/include/fuse_constraints/normal_prior_orientation_3d_euler_cost_functor.h

```diff
@@ -35,29 +35,28 @@
 #define FUSE_CONSTRAINTS_NORMAL_PRIOR_ORIENTATION_3D_EULER_COST_FUNCTOR_H
 
 #include <fuse_constraints/util.h>
+#include <fuse_variables/util.h>
 #include <fuse_variables/orientation_3d_stamped.h>
 
-#include <angles/angles.h>
 #include <ceres/internal/disable_warnings.h>
 #include <ceres/internal/eigen.h>
 #include <ceres/rotation.h>
 #include <Eigen/Core>
 
+#include <vector>
+
 
 namespace fuse_constraints
 {
 
 /**
- * @brief Create a prior cost function on a 3D orientation variable (quaternion)
- *
- * The Ceres::NormalPrior cost function only supports a single variable. This is a convenience cost function that
- * applies a prior constraint on a 3D orientation.
+ * @brief Create a prior cost function on a 3D orientation variable using Euler roll, pitch, and yaw measurements
  *
  * The cost function is of the form:
  *
- *             ||    [  roll - b(1)] ||^2
- *   cost(x) = ||A * [  pitch - b(2)] ||
- *             ||    [  yaw - b(3)] ||
+ *             ||    [  roll  - b(1)  ] ||^2
```

Comment:
Should probably zero-index the components. And it's a little deceiving because the order is defined by the `axes` parameter. I don't really have a suggestion on how to make it both readable _and_ not deceiving.

Maybe just by example?
```
The order of the measured axes is defined in the \p axes variable. For example, if the yaw and roll were measured in that order, then:
b_ = [ measured_yaw  ]
     [ measured_roll ]

cost(x) = || A * [ yaw(x)  - b_(0) ] ||^2
          ||     [ roll(x) - b_(1) ] ||

```